lib/music_library_controller.rb

```diff
@@ -0,0 +1,99 @@
+class MusicLibraryController
+  def initialize(path = "./db/mp3s")
+    MusicImporter.new(path).import
+    @view = MusicLibraryView.new
```

Comment:
#### Code Quality - Awesome
Good use of the MVC Pattern to keep concerns separate.